recipe/meta.yaml

```diff
@@ -162,11 +162,11 @@ requirements:
     - __glibc >=2.17      # [linux]
     - scipy >=1.7
     - optuna >=3
+    # TODO: Waiting for `aarch64` & `ppc64le` CUDA 12 support
     - {{ pin_compatible('cudnn') }}                        # [not ((aarch64 or ppc64le) and cuda_compiler_version == "12.0")]
-    # TODO: wait for NCCL to catch up
-    - {{ pin_compatible('nccl') }}                         # [linux and cuda_compiler_version != "12.0"]
+    - {{ pin_compatible('nccl') }}                         # [linux64 or ((aarch64 or ppc64le) and cuda_compiler_version != "12.0")]
     - {{ pin_compatible('cutensor', lower_bound='1.3') }}
-    - {{ pin_compatible('cusparselt', max_pin='x.x') }}    # [(linux64 or win) and cuda_compiler_version == "11.2"]
+    - {{ pin_compatible('cusparselt', max_pin='x.x') }}    # [linux64 or (win and cuda_compiler_version == "11.2")]
```

Comment:
Tried to add `linux_64` in a few places while excluding `linux_aarch64` & `linux_ppc64le`. Though this could probably use review

Reply:
I reverted the cuSPARSELt part because cuSPARSELt keeps breaking APIs and CuPy hasn't been able to catch up (https://github.com/cupy/cupy/issues/6757). We still need to use v0.2.0 here.